app/components/DocumentPreview/DocumentPreview.js

```diff
@@ -84,6 +84,8 @@ const Heading = styled.h3`
   margin-bottom: 0.25em;
   overflow: hidden;
   white-space: nowrap;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
```

Comment:
Went a bit overboard with the heading typeface, it shouldn't appear in list items